ironmon_tracker/network/Network.lua

```diff
@@ -326,24 +326,24 @@ function Network.getStreamerbotCode()
 end
 
 function Network.openUpdateRequiredPrompt()
-	local form = Utils.createBizhawkForm("Streamerbot Update Required", 350, 150, 100, 50)
+	local form = ExternalUI.BizForms.createForm("Streamerbot Update Required", 350, 150)
 	local x, y, lineHeight = 20, 20, 20
-	forms.label(form, string.format("Streamerbot Tracker Integration code requires an update."), x, y, 330, 20)
+	form:createLabel(string.format("Streamerbot Tracker Integration code requires an update."), x, y)
 	y = y + lineHeight
-	forms.label(form, string.format("You must re-import the code to continue using Stream Connect."), x, y, 330, 20)
+	form:createLabel(string.format("You must re-import the code to continue using Stream Connect."), x, y)
```

Comment:
Similar to other comment, should these labels be moved to the language files?

Reply:
Not in this PR, see other comment.